cg/cli/workflow/rnafusion/base.py

```diff
@@ -94,15 +67,14 @@ def start(
     revision: str,
     compute_env: str,
     use_nextflow: bool,
-    genomes_base: Path,
     dry_run: bool,
 ) -> None:
     """Start full workflow for CASE ID."""
     LOG.info(f"Starting analysis for {case_id}")
 
     analysis_api: RnafusionAnalysisAPI = context.obj.meta_apis["analysis_api"]
     analysis_api.prepare_fastq_files(case_id=case_id, dry_run=dry_run)
-    context.invoke(config_case, case_id=case_id, genomes_base=genomes_base, dry_run=dry_run)
+    context.invoke(config_case, case_id=case_id, dry_run=dry_run)
     context.invoke(
```

Comment:
We should not call other CLI commands in a CLI command. 
can these be replaces by:
```Python 
nf_analysis_api: NfAnalysisAPI = context.meta_apis[MetaApis.ANALYSIS_API]
nf_analysis_api.config_case(case_id=case_id, dry_run=dry_run)
nf_analysis_api.run_nextflow_analysis(
            case_id=case_id,
            dry_run=dry_run,
            log=log,
            work_dir=work_dir,
            from_start=from_start,
            profile=profile,
            config=config,
            params_file=params_file,
            revision=revision,
            compute_env=compute_env,
            use_nextflow=use_nextflow,
            nf_tower_id=nf_tower_id,
        )
```

Reply:
I like this. However I don't want to touch the `start` subcommand in this PR. I will add it in the next PR when add start for tomte, so I can test it properly there. But absolutely agreeing!